SPECS/bind/bind.spec

```
@@ -9,8 +9,8 @@
 
 Summary:        Domain Name System software
 Name:           bind
-Version:        9.16.33
-Release:        2%{?dist}
+Version:        9.16.37
+Release:        3%{?dist}
```

This should be `1` if we bump the version :)
```suggestion
Release:        3%{?dist}
```

---

Thank you for catching that!